modules/standard/IO.chpl

```diff
@@ -3899,6 +3953,107 @@ proc channel.readline(ref arg: ?t): bool throws where t==string || t==bytes {
   return true;
 }
 
+/*
+  Read a line into a Chapel string. Reads until a ``\n`` is reached.
+
+  :arg arg: a string to receive the line
+  :arg maxSize: The maximum amount of characters to read.
```

Comment:
Please have it say "the maximum number of codepoints to read"

Reply:
Also needs to say that "the default of -1 means no limit on the number of codepoints to read".